edb/server/net_worker.py

```diff
@@ -351,6 +351,7 @@ def _warn(e):
                 variables={"expires_in": expires_in.to_backend_str()},
                 cached_globally=True,
             )
+            result: list[dict] = json.loads(result_json)
             if len(result) > 0:
                 logger.info(f"Deleted requests: {result!r}")
```

Comment:
I don't think it's a good idea to log all requests here probably. The list could be considerably large.  Perhaps just log the count?

Reply:
Changed to log the count